dnf.spec

```diff
@@ -250,6 +250,13 @@ ln -sr  %{buildroot}%{confdir}/vars %{buildroot}%{_sysconfdir}/yum/vars
 rm %{buildroot}%{confdir}/%{name}.conf
 %endif
 
+%if 0%{?fedora} >= 41 || 0%{?rhel} >= 10
+%undefine _py3_shebang_P
```

Comment:
@ppisar Because DNF with the `-P` option does not work.
I removed the unwanted flag from the macro according to the Python Packaging Guidelines https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_shebangs

Reply:
What do you mean with "does not work"? Loading Python modules from current working directory is terribly dangerous. Could you report the "does not work with -P" with more details as a new issue?

Reply:
> What do you mean with "does not work"?

Result with `-sP`:
```
# dnf
Traceback (most recent call last):
  File "/usr/bin/dnf", line 61, in <module>
    from dnf.cli import main
  File "/usr/lib/python3.12/site-packages/dnf/__init__.py", line 30, in <module>
    import dnf.base
  File "/usr/lib/python3.12/site-packages/dnf/base.py", line 29, in <module>
    import libdnf.transaction
ImportError: dynamic module does not define module export function (PyInit_libdnf)
```